app/views/components/currency-input/index.html

```diff
@@ -0,0 +1,56 @@
+{% extends "layouts/base.html" %}
+
+{% block body %}
+
+  <div class="govuk-grid-row">
+
+    <div class="govuk-grid-column-full">
+
+      {{ govukBackLink({
+        text: "Back",
+        href: "../"
+      }) }}
+
+      <h1 class="govuk-heading-xl" tabindex="-1">
+        <span class="govuk-caption-xl">Components</span> Currency input
+      </h1>
+
+      {{ mojCurrencyInput({
+        id: "amount",
+        classes: "govuk-input--width-10",
+        name: "amount",
+        label: {
+          text: "Amount",
+          classes: 'govuk-!-font-weight-bold'
+        },
+        hint: {
+          text: "Enter the amount you want to exchange"
+        }
+      }) }}
+
+      {{ mojCurrencyInput({
+        id: "amount",
+        classes: "govuk-input--width-10",
+        name: "amount",
+        currencyLabel: {
+          text: "&yen;",
+          label: "Yen",
```

Comment:
I can't see where this is used in the codebase.

Reply:
I have removed the `label` param